testcontainers/src/core/image/image_ext.rs

```diff
@@ -54,6 +57,13 @@ pub trait ImageExt<I: Image> {
     /// Adds a mount to the container.
     fn with_mount(self, mount: impl Into<Mount>) -> ContainerRequest<I>;
 
+    /// Copies some source into the container as file
+    fn with_copy_to<P: AsRef<std::path::Path>>(
+        self,
+        target: P,
```

Comment:
Btw, I assume we shouldn't use `Path` as an argument for `target`. Let's say you're on Windows, but target is Unix-based container

Reply:
This is a very valid point. `String` it is then :)